tests/odr/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+file(GLOB_RECURSE FOUND_HEADERS "${CMAKE_CURRENT_SOURCE_DIR}/../../include/*.hpp")
+list(TRANSFORM FOUND_HEADERS REPLACE ".*/include/" "")
+list(FILTER FOUND_HEADERS EXCLUDE REGEX ".*dnsa_connection.*")
+
+foreach(header_file ${FOUND_HEADERS})
+  string(REGEX REPLACE "[/\\]" "-" source_file ${header_file})
+  get_filename_component(source_file ${source_file} NAME_WE)
+  configure_file(${CMAKE_CURRENT_SOURCE_DIR}/odr.tmpl
+                 ${CMAKE_CURRENT_BINARY_DIR}/${source_file}.1.odr.cpp)
+  configure_file(${CMAKE_CURRENT_SOURCE_DIR}/odr.tmpl
+                 ${CMAKE_CURRENT_BINARY_DIR}/${source_file}.2.odr.cpp)
+endforeach()
+
+file(GLOB FOUND_ODR_SOURCES "${CMAKE_CURRENT_BINARY_DIR}/*.cpp")
+
+add_library(test_odr SHARED ${FOUND_ODR_SOURCES})
+setup_test(test_odr)
```

Comment:
What does this step serve?

Reply:
It's a function that setups flags and links for target.
Result of: https://github.com/seznam/SuperiorMySqlpp/pull/129#discussion_r506471902

Reply:
I know - I was wondering whether all that's done there is really needed for ODR test. It isn't, but it's probably good enough tradeoff to keep it as it is.